client/blocks/app-banner/style.scss

```diff
@@ -91,10 +92,14 @@ body.app-banner-is-visible {
 
 	&.jetpack {
 		font-size: rem(28px); //typography-exception
-		font-weight: bold;
+		font-weight: 500;
 		max-width: 18em;
 		margin-left: auto;
 		margin-right: auto;
+		-webkit-font-smoothing: auto;
+		-moz-osx-font-smoothing: initial;
+		text-rendering: auto;
+		@extend .wp-brand-font;
```

Comment:
`@extend .wp-brand-font;` is the recommended way to apply the `$brand-serif` fonts to ensure a device's locale is taken into account ([reference](https://github.com/Automattic/wp-calypso/blob/trunk/packages/typography/README.md)).